test-apps/display-test-app/src/backend/DtaElectronMain.ts

```diff
@@ -32,8 +31,8 @@ const getWindowSize = () => {
 
 class DtaIpcImpl extends IpcHandler implements DtaIpcInterface {
   public get channelName() { return dtaChannel; }
-  public async openFile(options: OpenDialogOptions) {
-    return dialog.showOpenDialog(options);
+  public async sayHello() {

```

Comment:
Was this just for testing? Any  reason to keep it?

Reply:
I just figured at some point somebody is going to need to call a backend method from DTA and i wanted to keep this scaffolding to show how to do that. But our linter doesn't allow empty interfaces.